### ISSUE TITLE
Scrolling is horrible When one wants to read more than a few pages of unread messages, it is almost impossible to scroll through them slowly - because there are no end arrows at the scroll bar, and the locator on the bar jumps back up whenever new messages are loaded, causing a run because the mouse pointer now points way below the locator.

### ISSUE THREAD

**Issue by Biep** [vector-im/riot-web#8455]

**Comment by rubo77** [comment]
I agree, but you should change the title to just "Enhance scrolling in message history" and put your description in the first post (use the three dots to edit your first post)

**Comment by Biep** [comment]
Done.  I was convinced I had given two arguments to !github create - but seemingly I made some mistake with the quotes.

**Comment by lampholder** [comment]
I've been thinking a little about the scrollbar, and how I don't think it adds a lot of value to our present UX: https://github.com/vector-im/riot-web/issues/8065

#8065 doesn't have any answers to the problem you raise here, but I think the UX I'm imagining might be a decent match - if the traditional scrollbar UX were replaced with something that both represented _and_ afforded good control over your incremental- (rather than jump-) scrolling through the timeline.

**Comment by rubo77** [comment]
Also the jumping gets worse, if URL previews are enabled : once a preview is loaded it reorderes the content again!

**Comment by 3nprob** [comment]
Element is practically unusable for navigating in long histories. If I'm looking for something particular even just a day or two back in a busy channel, it's a lot more efficient to export 8MB of history and load up the export in a web browser. This should not be the case.
